app/presenters/finder_presenter.rb

```diff
@@ -118,6 +117,22 @@ def label_for_metadata_key(key)
 private
   attr_reader :content_item, :values
 
+  def part_of
+    content_item.links.part_of || []
+  end
+
+  def organisations
+    content_item.links.organisations || []
+  end
+
+  def people
+    content_item.links.people || []
+  end
+
+  def from
+    (organisations + people).compact
```

Comment:
You shouldn't need to `compact` this, as neither `organisations` nor `people` should be returning arrays including `nil`.